datadog/resource_datadog_dashboard_test.go

```diff
@@ -634,6 +684,130 @@ resource "datadog_dashboard" "free_dashboard" {
 }`, uniq)
 }
 
+func datadogSimpleFreeDashboardConfig(uniq string) string {
+	return fmt.Sprintf(`
+resource "datadog_dashboard" "simple_free_dashboard" {
```

Comment:
You need to keep the same resource name in your update, otherwise terraform ends doing a delete/create and you don't test forceNew

Reply:
🤦